api.c

```diff
@@ -57,7 +60,7 @@ struct api_ctx {
 
 struct api_ctx *api_ctx_alloc(struct slirp4netns_config *cfg)
 {
-    struct api_ctx *ctx = (struct api_ctx *)g_malloc0(sizeof(*ctx));
+    struct api_ctx *ctx = (struct api_ctx *)calloc(1, sizeof(*ctx));
```

Comment:
Why?

Reply:
g_malloc0 using malloc internally is just an implementation detail.  Since we free `ctx` with `free`, we should either change `g_malloc()` -> `malloc()`, or `free()` with `g_free()`.